File: InventoryManagement.Api/Contracts/ItemModel.cs
using System;

namespace InventoryManagement.Api.Contracts;

public class ItemModel
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Category { get; set; }
    public string Car { get; set; }
    public int Quantity { get; set; }
    public string Type { get; set; }
    public string Description { get; set; }
    public string Price { get; set; }


}

File: InventoryUI/inventory-ui/src/app/contracts/item.model.ts
export class Item {
    id: number;
    name: string;
    category: string;
    car: string;
    quantity: number;
    description: string;
    price: number;
  
    constructor(
      name: string,
      category: string,
      car: string,
      quantity: number,
      description: string,
      price: number,
      id: number
    ) {
      this.name = name;
      this.category = category;
      this.car = car;
      this.quantity = quantity;
      this.description = description;
      this.price = price;
      this.id = id;
    }
  }
  